src/components/FeaturedData/index.js

```diff
@@ -0,0 +1,248 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import dynamic from 'next/dynamic';
+import classNames from 'classnames';
+
+import Grid from '@material-ui/core/Grid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Typography from '@material-ui/core/Typography';
+import withApollo from 'lib/withApollo';
+
+import ChartFactory from '@codeforafrica/hurumap-ui/factory/ChartFactory';
+import visualLoader from '@codeforafrica/hurumap-ui/factory/utils/visualLoader';
+
+import { ThemeProvider } from '@material-ui/core/styles';
+
+import config from 'config';
+import logo from 'assets/images/logos/dominion-logo-small.png';
+import useChartDefinitions from 'data/useChartDefinitions';
+import featuredCharts from 'data/featuredCharts';
+
+const COLOR_SCALE = ['#696969', '#8D8D8C', '#A9A9A9', '#C0C0C0', '#D3D3D3'];
+const InsightContainer = dynamic(
+  () => import('@codeforafrica/hurumap-ui/core/InsightContainer'),
+  {
+    ssr: false
+  }
+);
+
+const useStyles = makeStyles(({ breakpoints, palette, typography }) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: palette.primary.light
+  },
+  layout: {
+    padding: '60px 0',
+    margin: '0 auto',
+    [breakpoints.up('sm')]: {
+      maxWidth: '66.59rem' // .75 of lg
+    },
+    [breakpoints.up('md')]: {
+      maxWidth: '81.3571429rem'
+    }
+  },
+  title: {
+    margin: '1.375rem 0',
+    [breakpoints.up('md')]: {
+      width: '51.125rem'
+    }
+  },
+  chartTitle: {
+    fontFamily: typography.fontFamily,
+    color: '#525e31',
+    fontWeight: 'bold',
+    fontSize: '0.929rem !important'
+  },
+  dullOrange: {
+    color: '#de9f3a'
+  },
+  sourceLink: {
+    fontSize: typography.caption.fontSize
+  },
+  descriptionWrapper: {
+    backgroundColor: '#f1f1ed',
+    padding: '3.42rem 0 0 !important',
+    marginTop: '0 !important',
+    '& div:first-child': {
+      display: 'none'
+    }
+  },
+  description: {
+    fontWeight: '500',
+    fontSize: '0.857rem',
+    lineHeight: '1.92',
+    marginLeft: '0 !important',
+    width: '70%'
+  },
+  chartContainer: {
+    backgroundColor: 'white'
+  },
+  hideHighlightGrid: {
+    display: 'none'
+  }
+}));
+
+function FeaturedData({ selectedCountry }) {
+  const classes = useStyles();
+  const theme = useTheme();
+  const greyChartTheme = {
+    ...theme,
+    chart: {
+      ...theme.chart,
+      colorScale: COLOR_SCALE,
+      pie: {
+        ...theme.chart.pie,
+        colorScale: COLOR_SCALE
+      },
+      area: {
+        ...theme.chart.area,
+        colorScale: COLOR_SCALE
+      },
+      group: {
+        ...theme.chart.group,
+        colorScale: COLOR_SCALE
+      },
+      bar: {
+        ...theme.chart.bar,
+        style: {
+          ...theme.chart.bar.style,
+          data: {
+            ...theme.chart.bar.style.data,
+            fill: COLOR_SCALE[0]
+          }
+        }
+      }
+    }
+  };
```

Comment:
This can also be a function in `lib` to reduce the size of this component.

Reply:
:+1: